checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -0,0 +1,68 @@
+function GetContentFromConfigFile {
+    [CmdletBinding()]
```

Comment:
Regarding this and why it is useful:
`In PowerShell scripts, CmdletBinding() is used as a parameter attribute to define a PowerShell function or script as an advanced function. It is commonly used when creating cmdlets or more complex functions that require advanced parameter features.`

So I guess we should use it for all of our functions, since they will be somewhat complex.